packages/rln-pcd/package.json

```diff
@@ -24,7 +24,9 @@
     "lint": "eslint \"**/*.ts{,x}\"",
     "test": "ts-mocha --exit test/**/*.spec.ts",
     "build": "tsup --platform=browser src/index.ts --out-dir ./dist/browser && tsup --platform=node src/index.ts --out-dir ./dist/node",
-    "dev": "tsup --platform=browser src/index.ts --watch",
+    "dev": "yarn dev:browser & yarn dev:node",
```

Comment:
doing an `&` here works, but if you ctrl+c the process from terminal, the one that was moved into the background isn't killed. i am not sure if you tried that. to run indefinite processes in parallel from the command line, there's this npm package called [concurrently](https://github.com/open-cli-tools/concurrently) which you can use to invoke both commands, and have both of them die on ctrl+c.